### DISCUSSION TITLE
Unable to login to Vercel  via Github

### DISCUSSION THREAD

**Discussion by mtariqbeg** [vercel/community#3788] [Account Issue] [answered]
### Question

I did get mail from Github that your account has been granted access tio a third party Vercel but when i try to login via vercel the message is "The user Account is Blocked"

Need help in this regard.

Muhammad Tariq Baig
Github Id: mtariqbeg

**Comment by amyegan** [comment, marked as answer]
Due to the sensitive nature of account information, we cannot safely discuss it in this public forum. For more info, please check this guide: [Why has my account or deployment been paused?](https://vercel.com/guides/why-is-my-account-deployment-blocked)
